### ISSUE TITLE
Display Keybinds with ?

### ISSUE THREAD

**Issue by sbdchd** [mbunkus/mo-git-blame#18]
I think it would be helpful to show the available keybinds, and their uses, by pressing <kbd>?</kbd> when mo-git-blame is enabled.

**Comment by mbunkus** [comment]
I agree, that would be helpful. I'm currently not spending time on mo-git-blame, though, so I won't implement this myself. I'd really appreciate a pull request for it, though :)
